.github/workflows/lint.yaml

```diff
@@ -0,0 +1,61 @@
+name: lint
+on:
+  pull_request:
+  push:
```

Comment:
**Question:** Is there any particular reason to run this workflow on anything but PRs? I'm assuming we're requiring changes to be made via PR, so if that's the case, we can require that these lint status checks pass before merge, at which point master/develop will be guaranteed to pass the checks. Running them on a schedule wouldn't be necessary for the same reason; and unless we're triggering this workflow via the GitHub API, the `workflow_dispatch` event won't be used.

Reply:
I've adjusted this to only run the workflow on PRs.